src/util/isChaosToken.ttslua

```diff
@@ -0,0 +1,31 @@
+do
```

Comment:
I don't think we're going to want a pattern with one function per file.  That's going to explode fast.  Should probably make this something like ChaosTokenUtils and attach the function to that

Reply:
Actually let's just make this TokenUtils or TokenChecker - we can reasonably expand it to isClue(), isDoom(), etc

Reply:
Done